circuit_puzzles/programs/recharge_win.clsp

```diff
@@ -3,14 +3,15 @@
     statutes_puzzle_hash  lineage_proof announcements
     (ttl current_timestamp treasury_coins
      p2_treasury_hash funding_coin_id treasury_minimum
-     my_coin_id)
+     my_coin_id target_puzzle_hash)
```

Comment:
my_coin_id not needed anywhere. can be removed, also as argument to deposit-to-treasury function